src/redist/targets/packaging/windows/clisdk/VS.Redist.Common.Net.Core.Toolset.nuspec

```diff
@@ -1,18 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
-        <id>VS.Redist.Common.Net.Core.SDK.x64</id>
+        <id>VS.Redist.Common.Net.Core.Toolset.$ARCH$</id>
         <version>1.0.0</version>
-        <title>VS.Redist.Common.Net.Core.SDK.x64</title>
+        <title>VS.Redist.Common.Net.Core.Toolset.$ARCH$</title>
         <authors>Microsoft</authors>
         <owners>Microsoft</owners>
         <licenseUrl>https://www.microsoft.com/net/dotnet_library_license.htm</licenseUrl>
         <projectUrl>https://github.com/dotnet/core-sdk</projectUrl>
         <requireLicenseAcceptance>true</requireLicenseAcceptance>
-        <description>Windows Installers (x64) for .Net Core SDK</description>
+        <description>Windows Installer MSI ($ARCH$) for .NET core SDK Toolset for internal Visual Studio build consumption</description>
         <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     </metadata>
     <files>
         <file src="$DOTNET_BUNDLE$" />
```

Comment:
This should be for MSIs right? We're not inserting the bundle anymore or did I miss something?

Reply:
@joeloff
Correct, this is a NuPkg for the 'Toolset' MSI

Reply:
Changing "DOTNET_BUNDLE" to a generic "PAYLOAD_FILES"